rules/do-you-ensure-compliance-and-governance-without-compromising-agility/rule.md

```diff
@@ -1,14 +1,15 @@
 ---
 type: rule
 title: Do you ensure compliance and governance without compromising agility?
-uri: do-you-ensure-compliance-and-governance-without-compromising-agility
+uri: ensure-compliance-and-governance-without-compromising-agility
```

Comment:
need to update folder name to match uri

Reply:
Done ✅ - PR for folder changes: https://github.com/SSWConsulting/SSW.Rules.Content/pull/8357